inference-engine/tests/ngraph_functions/src/roi_pooling.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+//
+
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<Node> makeROIPooling(const Output<Node>& input,
```

Comment:
If we introduce new constructor it should allow to avoid this builder.

Reply:
As agreed in internal discussion, supporting strong type for ROIPooling method will be addressed in a different ticket: 43566